catalog/core/catalog-core-standardframework/src/test/java/ddf/catalog/impl/CatalogFrameworkImplTest.java

```diff
@@ -1888,11 +1880,10 @@ public void testGetAllSiteNames() {
   public void testProviderUnavailableCreate() throws SourceUnavailableException {
     MockEventProcessor eventAdmin = new MockEventProcessor();
     MockMemoryProvider provider =
-        new MockMemoryProvider(
-            "Provider", "Provider", "v1.0", "DDF", new HashSet<ContentType>(), false, null);
+        new MockMemoryProvider("Provider", "Provider", "v1.0", "DDF", new HashSet<>(), false, null);
```

Comment:
✏️ Provider and v1.0 could be made constants

Reply:
I made them constants.